bokehjs/src/less/main.less

```diff
@@ -153,11 +11,3 @@
   border: 1px dashed green;
   z-index: 100;
 }
-
-.bk-toolbar-button.hover:focus {
-  outline: none;
-}
-
-.box .bk-grid-column {
```

Comment:
@birdsarah, was there any use for this? Maybe this should be `bk-box`? Though, `bk-box` being present in another place (also removed in this PR), isn't used across the repository.